README.md

```diff
@@ -59,6 +59,7 @@ Name     | Options     | Default      | Description
 `editable`         | *boolean*    | false        | Switches item into editing mode (allows changing label)
 `placeholder`         | *string*    | Type item name here        | Label input placeholder, applies only if the item is in editing mode
 `checkboxHidden`         | *boolean*    | false        | Determines if checkbox is available for editing
+`listItemStyle`         | *string*    | 'number'        | Checkbox item style. Support 'number' or 'bullet'
```

Comment:
I think that we should change it to `listStyleType` as it's in CSS. Possible values should be:
- none (no list indicator at all)
- decimal (current number)
- disc (current bullet)

Then it will be consistent with CSS. Also please change description to: "List item indicator". Available values should be in "Options" column. See other juicy webcomponents.